ui/static/ui/js/listing.js

```diff
@@ -1,6 +1,75 @@
 define(['jquery', 'bootstrap', 'icheck', 'csrf'], function(jQuery) {
   'use strict';
 
+  /* global $ */
+
+  function formatGroupName(string) {
+    string = string.charAt(0).toUpperCase() + string.slice(1);
+    return string.substring(0, string.length - 1);
+  }
+
+  function isMITEmail(kerberosId) {
+    var email = kerberosId + '@mit.edu';
+    var regex = /^([a-zA-Z0-9_.+-])+\@(([a-zA-Z0-9-])+\.)+([a-zA-Z0-9]{2,4})+$/;
+    return regex.test(email);
+  }
```

Comment:
This currently doesn't check if the email is a valid MIT email.  If this is going to grow up to check whether the name is a valid Kerberos account, please add that as a comment.  Otherwise the name of the function is somewhat misleading.